packages/metal-isomorphic/package.json

```diff
@@ -0,0 +1,17 @@
+{
+	"name": "metal-isomorphic",
+	"version": "2.13.2",
+	"private": true,
+	"description": "Integration tests for server side rendering.",
+	"license": "BSD",
+	"repository": "https://github.com/metal/metal.js/tree/master/packages/metal-isomorphic",
+	"engines": {
+		"node": ">=0.12.0",
+		"npm": ">=3.0.0"
+	},
+	"dependencies": {
+		"metal-component": "^2.13.2",
+		"metal-jsx": "^2.13.2",
+		"metal-soy": "^2.13.2"
+	}
```

Comment:
Aren't we missing `devDependencies` here? (I know it's probably picked by lerna, but it feels weird, not sure if that's how it's supposed to be :) )

Reply:
Well, since the tests aren't actually run from this package, it will only add bloat to `lerna bootstrap` since the `devDependencies` in this package wouldn't be used. I can see why it looks a little confusing though, do you want to add `mocha` and the other test dependencies here?

Reply:
Sounds good, if that's what we're also doing in other projects... we need to converge all the monorepos' infrastructure :)